fedimint-bitcoind/src/lib.rs

```diff
@@ -50,6 +50,8 @@ pub trait IBitcoindRpc: Debug + Send + Sync + 'static {
     /// Returns a proof that a tx is included in the bitcoin blockchain
     async fn get_txout_proof(&self, txid: Txid) -> Result<TxOutProof>;
 
+    async fn get_info(&self) -> Result<(u64, bool)>;
```

Comment:
[nit] Trait docs (`Result<(u64, bool)>` isn't obvious at a glance)

Reply:
Fixed. Moved `BlockchainInfo` to `fedimint-bitcoind`